src/libraries/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/AtomPub10ServiceDocumentFormatterTests.cs

```diff
@@ -944,7 +943,7 @@ public void CreateItemInstance_Generic_Success()
             Assert.Empty(document.Workspaces);
 
             var typedFormatter = new GenericFormatter<ServiceDocumentSubclass>();
-            ServiceDocumentSubclass item = Assert.IsType<ServiceDocumentSubclass>(typedFormatter.CreateDocumentInstanceEntryPoint());
+            Assert.IsType<ServiceDocumentSubclass>(typedFormatter.CreateDocumentInstanceEntryPoint());
```

Comment:
Again, this looks like it was copied from the formatter/document checking code above. The checks here should be for "item" instead of re-checking "document".

Reply:
Updated!